executor/concurrent_map_test.go

```diff
@@ -16,19 +16,20 @@ package executor
 
 import (
 	"sync"
+	"testing"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/util/chunk"
+	"github.com/stretchr/testify/require"
 )
 
 // TestConcurrentMap first inserts 1000 entries, then checks them
-func (cm *pkgTestSuite) TestConcurrentMap(c *C) {
+func TestConcurrentMap(t *testing.T) {
 	m := newConcurrentMap()
```

Comment:
```suggestion
func TestConcurrentMap(t *testing.T) {
	t.Parallel()
	m := newConcurrentMap()
```

Run in parallel in a `xxx_test.go`. Also this test seems can be run in parallel.

Reply:
Wait for this change :)

Reply:
done

Reply:
@xhebox @tiancaiamao could you please give it another round to review?